File: frontend/src/app/models/team/player-statistics.response.ts
export interface PlayerStatisticsResponse {
  playerId: number;
  playerName: string;
  matchesPlayed: number;
  minutesPlayed: number;
  goals: number;
  yellowCards: number;
  redCards: number;
  substitutions: number;
  otherEvents: number;
  totalEvents: number;
}

File: src/main/java/com/tournament/app/footycup/backend/dto/team/PlayerStatisticsResponse.java
package com.tournament.app.footycup.backend.dto.team;

public record PlayerStatisticsResponse(
        Long playerId,
        String playerName,
        Integer matchesPlayed,
        Integer minutesPlayed,
        Integer goals,
        Integer yellowCards,
        Integer redCards,
        Integer substitutions,
        Integer otherEvents,
        Integer totalEvents
) {
}
